### DISCUSSION TITLE
Getting notified that a subscription is completed

### DISCUSSION THREAD

**Discussion by leifriksheim** [urql-graphql/urql#3592] [Q&A]
We are using the old `subscriptions-transport-ws` package for the SubscriptionClient as we are running a python backend with an outdated protocol implementation. We are struggling to find a way to understand when a subscription has finished. We can see the last incoming message being:

```
{"type":"complete","id":"1"}
```

But we have no way of knowing when this message is recieved from within urql using React and the hooks. I would imagine fetching, should be set to false when complete is done, or something similar. Anyone knows how to hook into this somehow?

**Comment by Sytten** [comment]
Did you ever figure it out @leifriksheim ?
